modules/multi-finality-verifier/src/lib.rs

```diff
@@ -99,7 +101,9 @@ pub mod pallet {
 
 		/// Weights gathered through benchmarking.
 		type WeightInfo: WeightInfo;
-	}
+
+        type TimeProvider: UnixTime;
```

Comment:
Not needed - re-use time from `EscrowTrait`, like `<T as EscrowTrait>::Time::now()`